### PR TITLE
Add without function

### PR DESCRIPTION
**Pull request by dzinrai**
without([2, 1, 2, 3], 1, 2)

// => [3]

### CODE REVIEW

**Comment by dzinrai** [comment]
#28

**Comment by dzinrai** [comment]
https://github.com/AllDayAlone/summer/issues/28

File: src/without.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function without(arr, ...values) {
+    function notExist(value) {
+        return ![...values].includes(value);
+    }
+    if (!arr.length) return [];
```
Comment (AllDayAlone) [inline src/without.js:5]: what if arr = null

File: src/without.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function without(arr, ...values) {
+    function notExist(value) {
+        return ![...values].includes(value);
```
Comment (AllDayAlone) [inline src/without.js:3]: just` !values.includes(value);`

File: src/without.js (hunk 1 above)
Comment (dzinrai) [reply src/without.js]: Yep https://github.com/AllDayAlone/summer/pull/37/commits/bc10feb0ce6b81287810fb213d01541e694a11a3

**Pull request merged**